engine/version/version_controller.hpp

```diff
@@ -70,6 +70,63 @@ class SnapshotList {
 // The snapshots include temporal snapshots that cached by each access thread of
 // kvdk instance, and a global snapshot list that actively created by user
 class VersionController {
+ public:
+  class LocalToken {
```

Comment:
1. I think LocalSnapshotHolder is more clear.
2. If use this to hold snapshot, please modify other places too.

Reply:
Let's discuss how to rename it.